utilities/make_agasc_supplement.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Create the initial version of agasc_supplement.h5.
+
+This file is a supplement to the stable AGASC to inform star selection
+and star catalog checking.
+
+This script simply creates the initial file that has only bad
+stars from two sources:
+ - starcheck bad star list
+   https://github.com/sot/starcheck/blob/master/starcheck/data/agasc.bad
+ - GAIA high proper motion file $SKA/analysis/gaia/agasc_gaia_xmatch_PM_gt_50mas.fits.gz
+   See: https://nbviewer.jupyter.org/url/cxc.cfa.harvard.edu/mta/ASPECT/ipynb/star_selection/gaia
+        GAIA guide star crossmatch.ipynb
+
+"""
+import os
+from pathlib import Path
+
+import numpy as np
+
+from astropy.table import Table
+
+HOME = Path(os.environ['HOME'])
+SKA = Path(os.environ['SKA'])
+
+agasc_ids = []
+sources = []
+
+# Starcheck bad star list is not installed anywhere so just grab from local git repo
+lines = open(HOME / 'git' / 'starcheck' / 'starcheck' / 'data' / 'agasc.bad', 'r').readlines()
```

Comment:
I'd just argue that this somewhat contaminates the new file with this list that is made half of manually added stars and half from star stats.

Reply:
I'm not very concerned about provenance and perfection at this point.  I just consider `agasc.bad` as a legacy source of bad stars with some recent manual additions and a 4-year old list of auto additions.  It's a starting point.

Going forward there will be a new process which takes some ideas from bad_stars.py but is mostly new.

Reply:
That's fine.  I just thought it would be as useful to import from the manual star file in that same directory and then add the rest of the stars that we're using in proseco / agasc.bad with a new source identifier.  But if you don't care, you don't care.